### DISCUSSION TITLE
Transfer穿梭框 如何实现  树形搜索

### DISCUSSION THREAD

**Discussion by hihuangwei** [ant-design/ant-design#41605] [General]
### What problem does this feature solve?

Transfer穿梭框 如何实现 树形搜索
https://ant.design/components/transfer-cn#components-transfer-demo-tree-transfer

**Comment by qidasheng369** [comment]
左右穿梭可以是自定义的树结构

**Reply by hihuangwei** [reply]
搜索结果是可以在左上角显示数量的，但是获取不到搜索结果的列表
